docs/topics/proc_configuring-jenkins-freestyle-project-to-deploy-an-application-with-fmp.adoc

```diff
@@ -0,0 +1,46 @@
+[id='configuring-jenkins-freestyle-project-to-deploy-your-application-fmp-{context}']
```

Comment:
Please do not deviate from the title, use `configuring-a-jenkins-freestyle-project-to-deploy-your-application-with-the-fabric8-maven-plugin_{context}` as the ID.